src/scripts/hc_webhooks.js

```diff
@@ -0,0 +1,32 @@
+import { User, connect } from 'human-connection-api-nodejs-client';
+import mongoose from 'mongoose';
+import slugify from 'slugify';
+import constants from '../config/constants';
+
+export async function contributeProcedure(data) {
+  connect(constants.HC_BACKEND_URL);
+  const ProcedureModel = mongoose.model('Procedure');
+  const { procedureId, email, password } = data || {};
+  if (procedureId && email && password) {
+    const procedure = await ProcedureModel.findOne({ procedureId });
+    if (procedure) {
+      const user = new User({ email, password });
+      await user.contribute({
+        title: procedure.title,
+        content: `<p>${procedure.abstract}</p>`,
+        contentExcerpt: procedure.abstract,
+        type: 'post',
+        language: 'de',
+        categoryIds: ['5ac7768f8d655d2ee6d48fe4'], // politics & democracy
+      }, {
+        slug: slugify(procedure.title, { lower: true }),
+        organization: constants.HC_ORGANIZATION_SLUG,
```

Comment:
leave organization empty, instead provide an organizationId field with the correct organization id. Or is this working correctly? Would surprize me a bit.

Reply:
The second param is `options` specific to HC library. It checks an existing organization with that slug beforehand